infra/self-hosted/entrypoint.sh

```
@@ -0,0 +1,37 @@
+#!/bin/sh
+set -e
+
+# Auto-generate AUTH_SECRET if not provided
+if [ -z "$AUTH_SECRET" ]; then
+  if [ -f /data/.auth_secret ]; then
+    export AUTH_SECRET=$(cat /data/.auth_secret)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)